e2e/cypress/integration/messaging/local_date_time_spec.js

```diff
@@ -49,46 +52,82 @@ describe('Messaging', () => {
+                    {postIndex: 1, standard: '12:30 PM', military: '12:30'},
+                    {postIndex: 2, standard: '8:30 PM', military: '20:30'},
+                    {postIndex: 3, standard: '12:30 AM', military: '00:30'},
                 ]
             },
             {
-                name: 'in other locale like Spanish',
+                name: 'in Spanish',
                 channelIntro: 'Inicio de Channel Test',
                 locale: 'es',
                 manualTimezone: 'UTC',
                 localTimes: [
-                    {postIndex: 0, match12hour: /4:30 a\. m\./, match24hour: '4:30'},
-                    {postIndex: 1, match12hour: /12:30 p\. m\./, match24hour: '12:30'},
-                    {postIndex: 2, match12hour: /8:30 p\. m\./, match24hour: '20:30'},
+                    {postIndex: 0, standard: '4:30 a. m.', military: '4:30'},
+                    {postIndex: 1, standard: '12:30 p. m.', military: '12:30'},
+                    {postIndex: 2, standard: '8:30 p. m.', military: '20:30'},
+                    {postIndex: 3, standard: '12:30 a. m.', military: '0:30'},
                 ]
             },
             {
-                name: 'for react-intl unsupported timezone',
+                name: 'in react-intl unsupported timezone',
                 channelIntro: 'Inicio de Channel Test',
                 locale: 'es',
                 manualTimezone: 'NZ-CHAT',
                 localTimes: [
-                    {postIndex: 0, match12hour: re12Hour, match24hour: re24Hour},
-                    {postIndex: 1, match12hour: re12Hour, match24hour: re24Hour},
-                    {postIndex: 2, match12hour: re12Hour, match24hour: re24Hour},
+                    {postIndex: 0, standard: '06:15 PM', military: '18:15'},
+                    {postIndex: 1, standard: '02:15 AM', military: '02:15'},
+                    {postIndex: 2, standard: '10:15 AM', military: '10:15'},
+                    {postIndex: 3, standard: '02:15 PM', military: '14:15'},
                 ]
             },
-        ].forEach((testCase) => {
-            it(testCase.name, () => {
-                verifyLocalizeTime(testCase);
+        ];
+
+        testCases.forEach((testCase) => {
+            describe(testCase.name, () => {
+                describe('standard time', () => {
+                    testCase.localTimes.forEach((localTime, index) => {
+                        it('post ' + index + ' should match', () => {
+                            // # Change user preference to 12-hour format
+                            setTo24HourTimeFormat(false);
+
+                            // # Set user locale and timezone
+                            setLocaleAndTimezone(testCase.locale, testCase.manualTimezone);
+
+                            // * Verify that the channel is loaded correctly based on locale
+                            cy.findByText(testCase.channelIntro).should('be.visible');
```

Comment:
This is failing on me due to channel intro being out of view partially.  I'd suggest to increase viewport height by `cy.viewport(1300, 800)` before each test. (We're currently using 600 height for testing but maybe we'll change this in the future as I'm seeing unnecessary "scrollIntoView" action).

Reply:
Fixed!